test/unit/validatorFunctions.test.js

```diff
@@ -0,0 +1,99 @@
+var expect = require('chai').expect,
+  SchemaUtils = require('../../lib/schemaUtils');
+describe('Validator function tests', function () {
+  describe('checkRequestBody function', function () {
+    var requestBody = {
+        mode: 'raw',
+        raw: JSON.stringify({
+          status: 200,
+          result: true
+        })
+      },
+      schemaPath = {
+        requestBody: {
+          content: {
+            'application/json': {
+              schema: {
+                properties: {
+                  status: {
+                    type: 'integer'
+                  },
+                  result: {
+                    type: 'boolean'
+                  },
+                  message: {
+                    type: 'string'
+                  }
+                },
+                required: ['status', 'result', 'message']
+              }
+            }
+          }
+        }
+      };
+
+    it('should return empty mismatch array if validationPropertiesToIgnore option includes BODY', function () {
+      SchemaUtils.checkRequestBody({}, '$.request.body', '$.paths[/path1].get', {}, {}, {
```

Comment:
this test will pass even if the valiationProperties option isn't passed, right? you'll need to pass a schema and a request with  a wrong body for the test to make sense

Reply:
No. In the checkRequestBody function, we directly check for `valiationPropertiesToIgnore.includes['BODY']`, without even validating it to be present.